doc/man3/EVP_set_default_properties.pod

```diff
@@ -27,10 +30,22 @@ given with I<libctx> (NULL signifies the default library context).
 Any previous default property for the specified library context will
 be dropped.
 
+EVP_default_properties_enable_fips() sets the 'fips=yes' to be a default property
+if I<enable> is non zero, otherwise it clears 'fips=yes' as a default property
+for the given I<libctx>. It merges the fips default property setting with any
+existing query strings that have been set via EVP_set_default_properties().
```

Comment:
Is it worth a note indicating that the change replaces any 'fips' query element that was there or is this sufficiently implied?

Reply:
I thought that was what merges implies